UnityProject/Assets/Scripts/Systems/Explosions/Explosion.cs

```diff
@@ -93,6 +97,7 @@ public void Step()
 					NodePoint.ExplosionNode.Initialise(Local, Matrix.Matrix);
 				}
 
+				NodePoint.ExplosionNode.isEMP = isEMP;
```

Comment:
one issue here is that if you overlap 2 explosions, the EMP will be multiplied by the power of the second explosion even though that's a non EMP or the EMP effect Might get removed

Reply:
I suppose Im wrong, but after reading through explosion manager code it seems like both of the overlapping explosions will just add their nodes into CheckLocations and they wont affect each other. Could you please explain what exactly can go wrong?

Reply:
so, Check locations adds up all the explosions that happened on a node, before it was processed last, and then does the effect then, this means that overlapping explosions will have unpredicted behaviour with isEMP

Reply:
the proper way to do would be  adding a EMP  strength  float on  (node) tile and then   instead of  check  explosion resistance you check EMP  resistance, but for now I think it's ok

Reply:
Might update it once emerged

Reply:
explosion resistance?

Reply:
EnergyExpended = metaTileMap.ApplyDamage(v3int, Damagedealt,
uses  armour of the tile to work out how much it exhorts
Will be a bit of a pain so you can skip doing the EMP resistance, but I do recommend making a separate variable for emp strength on tile (node)

Reply:
ik this is very dumb solution but I think its just simpler to create second bool which states that this node is also explosion so it apllies both emp and regular effects. I tested it with two overlapping explosions, works fine

Reply:
ok, made it with emp strength as separate value